README.md

```diff
@@ -1,24 +1,48 @@
-# README
+# Junk4Dollars
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Build Status](https://travis-ci.com/riccjohn/junk4dollars-api.svg?branch=master)](https://travis-ci.com/riccjohn/junk4dollars-api)
 
-Things you may want to cover:
+# Installation
 
-* Ruby version
+## Rails
 
-* System dependencies
+Install `rails 6.0.0`
 
-* Configuration
+```zsh
+gem install rails -v 6.0.0
+```
 
-* Database creation
+Clone this repo
 
-* Database initialization
+```zsh
+git clone git@github.com:riccjohn/junk4dollars-api.git
+```
 
-* How to run the test suite
+Install dependencies
```

Comment:
It'd be good to name the database as a dependency as well

Reply:
👍

Reply:
@ijpiantanida do you just mean adding it to my 'Install Dependencies' section?

Reply:
Just mentioning that you need postgres installed and runnning